src/FlipMove.js

```diff
@@ -553,19 +561,24 @@ class FlipMove extends Component<ConvertedProps, FlipMoveState> {
 
     this.parentData.boundingBox = this.props.getPosition(parentDomNode);
 
+    // Splitting DOM reads and writes to be peformed in batches
+    const childrenBoundingBoxes = [];
+
     this.state.children.forEach(child => {
       const childKey = getKey(child);
 
       // It is possible that a child does not have a `key` property;
       // Ignore these children, they don't need to be moved.
       if (!childKey) {
+        childrenBoundingBoxes.push();
```

Comment:
Nit: can we change this to `childrenBoundingBoxes.push(null);`? This could just be a subjective thing but I find it a bit clearer if we explicitly push a falsy value, rather than implicitly pushing `undefined`.